lib/src/main/java/com/diffplug/spotless/java/ImportOrderStep.java

```diff
@@ -35,33 +35,54 @@
 import com.diffplug.spotless.FormatterStep;
 
 public final class ImportOrderStep {
-	// prevent direct instantiation
-	private ImportOrderStep() {}
+	private final String lineFormat;
```

Comment:
For the foreseeable future, I can only imagine two usages, `import %s` and `import %s;`.  How about this API, to help the code read a little more fluently, and also to keep the implementation a little more private.

```java
  private ImportOrderStep(String lineFormat) {
    this.lineFormat = lineFormat;
  }

  public static ImportOrderStep withSemicolon() {
    return new ImportOrderStep("import %s;");
  }

  public static ImportOrderStep noSemicolon() {
    return new ImportOrderStep("import %s");
  }  
```